{
   "active_lock_reason" : null,
   "assignee" : null,
   "assignees" : [],
   "author_association" : "CONTRIBUTOR",
   "body" : "### Is there an existing issue for this?\r\n\r\n- [X] I have searched the existing issues\r\n\r\n### Current behaviour\r\n\r\nCI test failure https://cirrus-ci.com/task/5315879898972160?logs=functional_tests#L2675:\r\n\r\n```\r\ntest  2023-03-23T17:23:41.645000Z TestFramework (ERROR): JSONRPC error \r\nTraceback (most recent call last):\r\n  File \"C:\\Users\\ContainerAdministrator\\AppData\\Local\\Temp\\cirrus-ci-build\\test\\functional\\test_framework\\test_framework.py\", line 132, in main\r\n    self.run_test()\r\n  File \"C:\\Users\\ContainerAdministrator\\AppData\\Local\\Temp\\cirrus-ci-build\\test\\functional\\wallet_create_tx.py\", line 35, in run_test\r\n    self.test_create_too_long_mempool_chain()\r\n  File \"C:\\Users\\ContainerAdministrator\\AppData\\Local\\Temp\\cirrus-ci-build\\test\\functional\\wallet_create_tx.py\", line 98, in test_create_too_long_mempool_chain\r\n    tx_data = test_wallet.send(outputs=[{test_wallet.getnewaddress(): 25 - i}], options=options)\r\n  File \"C:\\Users\\ContainerAdministrator\\AppData\\Local\\Temp\\cirrus-ci-build\\test\\functional\\test_framework\\coverage.py\", line 49, in __call__\r\n    return_val = self.auth_service_proxy_instance.__call__(*args, **kwargs)\r\n  File \"C:\\Users\\ContainerAdministrator\\AppData\\Local\\Temp\\cirrus-ci-build\\test\\functional\\test_framework\\authproxy.py\", line 149, in __call__\r\n    raise JSONRPCException(response['error'], status)\r\ntest_framework.authproxy.JSONRPCException: Not solvable pre-selected input COutPoint(e425cbbd63, 1) (-4)\r\n```\r\n\r\n(log also includes a `feature_config_args` failure not related to this issue)\r\n\r\n### Expected behaviour\r\n\r\nTest should pass. This seems like a race condition as there is an `AddToWallet` log line immediately below the exception.\r\n\r\n### Steps to reproduce\r\n\r\nHappened in CI on commit 53d99551e913bfb85769a2b34fed73898779ff0f, but probably does not happen reliably.\r\n\r\n### Relevant log output\r\n\r\nhttps://cirrus-ci.com/task/5315879898972160?logs=functional_tests#L2675\r\n\r\n### How did you obtain Bitcoin Core\r\n\r\nOther\r\n\r\n### What version of Bitcoin Core are you using?\r\n\r\nmaster@ a70911492fcd1f726782cca2f1ffac18c25e8fb4\r\n\r\n### Operating system and version\r\n\r\nWindows\r\n\r\n### Machine specifications\r\n\r\n_No response_",
   "closed_at" : "2023-03-28T14:41:26Z",
   "closed_by" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/863730?v=4",
      "events_url" : "https://api.github.com/users/fanquake/events{/privacy}",
      "followers_url" : "https://api.github.com/users/fanquake/followers",
      "following_url" : "https://api.github.com/users/fanquake/following{/other_user}",
      "gists_url" : "https://api.github.com/users/fanquake/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/fanquake",
      "id" : 863730,
      "login" : "fanquake",
      "node_id" : "MDQ6VXNlcjg2MzczMA==",
      "organizations_url" : "https://api.github.com/users/fanquake/orgs",
      "received_events_url" : "https://api.github.com/users/fanquake/received_events",
      "repos_url" : "https://api.github.com/users/fanquake/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/fanquake/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/fanquake/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/fanquake"
   },
   "comments" : 4,
   "comments_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/27316/comments",
   "created_at" : "2023-03-23T18:21:41Z",
   "events_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/27316/events",
   "html_url" : "https://github.com/bitcoin/bitcoin/issues/27316",
   "id" : 1638072746,
   "labels" : [
      {
         "color" : "08a781",
         "default" : false,
         "description" : null,
         "id" : 149424,
         "name" : "Wallet",
         "node_id" : "MDU6TGFiZWwxNDk0MjQ=",
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/labels/Wallet"
      },
      {
         "color" : "d4c5f9",
         "default" : false,
         "description" : null,
         "id" : 62963516,
         "name" : "Tests",
         "node_id" : "MDU6TGFiZWw2Mjk2MzUxNg==",
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/labels/Tests"
      }
   ],
   "labels_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/27316/labels{/name}",
   "locked" : false,
   "milestone" : null,
   "node_id" : "I_kwDOABII585howGq",
   "number" : 27316,
   "performed_via_github_app" : null,
   "reactions" : {
      "+1" : 0,
      "-1" : 0,
      "confused" : 0,
      "eyes" : 0,
      "heart" : 0,
      "hooray" : 0,
      "laugh" : 0,
      "rocket" : 0,
      "total_count" : 0,
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/27316/reactions"
   },
   "repository_url" : "https://api.github.com/repos/bitcoin/bitcoin",
   "state" : "closed",
   "state_reason" : "completed",
   "timeline_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/27316/timeline",
   "title" : "wallet_create_tx.py \"Not solvable pre-selected input\" exception",
   "updated_at" : "2023-03-28T14:41:26Z",
   "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/27316",
   "user" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/7133040?v=4",
      "events_url" : "https://api.github.com/users/ryanofsky/events{/privacy}",
      "followers_url" : "https://api.github.com/users/ryanofsky/followers",
      "following_url" : "https://api.github.com/users/ryanofsky/following{/other_user}",
      "gists_url" : "https://api.github.com/users/ryanofsky/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/ryanofsky",
      "id" : 7133040,
      "login" : "ryanofsky",
      "node_id" : "MDQ6VXNlcjcxMzMwNDA=",
      "organizations_url" : "https://api.github.com/users/ryanofsky/orgs",
      "received_events_url" : "https://api.github.com/users/ryanofsky/received_events",
      "repos_url" : "https://api.github.com/users/ryanofsky/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/ryanofsky/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/ryanofsky/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/ryanofsky"
   }
}
